### PR TITLE
Emin/4

### PR DESCRIPTION
**Pull request by eminaydin**
Test was not working because of missing features in package.json. Now it works without any problem.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,15 +4,20 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
-    "dev": "nodemon ./bin/www"
+    "dev": "nodemon ./bin/www",
+    "test": "jest"
   },
   "dependencies": {
+    "@hapi/joi": "^17.1.0",
```
Comment (oliverwebr) [inline package.json]: Not part of this issue

File: routes/middleware/ThirdTask.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const Joi = require('@hapi/joi');
```
Comment (oliverwebr) [inline routes/middleware/ThirdTask.js]: Not part of this issue

File: routes/middleware/firstTask.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+let myFunc = function (req, res, next) {
+    console.log(req.method +  " on " + req.originalUrl );
+    console.log(req.body);
+    
+    
+    next();
+}
+
+
+module.exports = myFunc;
```
Comment (oliverwebr) [inline routes/middleware/firstTask.js]: Not part of this issue

File: routes/middleware/secondTask.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+const Joi = require('@hapi/joi');
```
Comment (oliverwebr) [inline routes/middleware/secondTask.js]: not part of this issue

File: routes/students.js [hunk 5]
```diff
@@ -1,6 +1,8 @@
 const express = require("express");
+const validation = require("./middleware/secondTask");
```
Comment (oliverwebr) [inline routes/students.js]: Not part of this issue

File: test/test.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+const request = require("supertest");
+const app = require("../app");
+
+describe("Test the students route", () => {
+  test("It should response the GET method", async () => {
+    const response = await request(app).get("/api/students");
+    expect(response.statusCode).toBe(200);
+
+  });
```
Comment (oliverwebr) [inline test/test.js]: Awesome

File: package.json (hunk 1 above)
Comment (eminaydin) [reply package.json]: What do you mean? Shall I just delete it?

Comment (oliverwebr) [reply package.json]: Jap

Comment (eminaydin) [reply package.json]: How can I push only one file instead of my whole folder?

Comment (oliverwebr) [reply package.json]: Don't add the files you don't want to push. We will talk more about this in the outro

**Pull request closed without merging**